text/plain.go

```diff
@@ -17,7 +17,7 @@ type Plain struct{}
 var _ Handler = (*Plain)(nil)
 
 // Lines splits a given block of text into separate lines.
-func (hdlr Plain) Lines(txt string) []string {
+func (hdlr *Plain) Lines(txt string) []string {
```

Comment:
Why do this?

Reply:
this is a remnant of a previous thinking/design for the "font cache" that was designed to be held by value (instead of by pointer-to-value as of right now).
this was made to be sure modifications applied to the font cache (and thru the `text.Plain` handler) were persisted.

I'll revert to a non-ptr-receiver.

Reply:
done.

Reply:
done.